plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -231,6 +231,23 @@ func (r *NodeAuthorizer) authorize(nodeName string, startingType vertexType, att
 	return authorizer.DecisionAllow, "", nil
 }
 
+// authorizeServiceAccount authorizes
+// - "get", "list", "watch" requests to serviceaccounts when KubeletServiceAccountTokenForCredentialProviders feature is enabled
+// - "create" requests to serviceaccounts 'token' subresource of pods running on a node
+func (r *NodeAuthorizer) authorizeServiceAccount(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
+	verb := attrs.GetVerb()
+	switch verb {
+	case "get", "list", "watch":
```

Comment:
`get` is not needed AFAICT.

```suggestion
	case "list", "watch":
```